lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -41,11 +42,35 @@ sealed class Transaction {
     abstract val type: TransactionType
 }
 
+@OptIn(ExperimentalSerializationApi::class)
+@Serializable
+data class TransactionHashPair(
+    @Serializable(with = TransactionPolymorphicSerializer::class)
+    @SerialName("transaction")
+    @JsonNames("txn")
+    val transaction: Transaction,
+
+    @SerialName("transaction_hash")
+    @JsonNames("txn_hash")
+    val hash: Felt,
+)
+
 @OptIn(ExperimentalSerializationApi::class)
 @Serializable
 @SerialName("DEPLOY")
 // OptIn needed because @JsonNames is part of the experimental serialization api
 data class DeployTransaction(
+    // not in RPC spec
+    @SerialName("transaction_hash")
+    @JsonNames("txn_hash")
+    override val hash: Felt = Felt.ZERO,
```

Comment:
I don't think this is right, and might lead to all kinds of problems

Reply:
\+ please do not change the order of properties, it is really hard ro review other changes :(

Reply:
Yup, agree. Please return to the old order 🙏